src/Subscribers/DoctrineEncryptSubscriber.php

```diff
@@ -175,19 +167,14 @@ public function postLoad(LifecycleEventArgs $args)
      */
     public function getSubscribedEvents()
```

Comment:
Ah didn't see that. Would require a new major version though. And would put 7.0 users out in the cold.